packages-next/fields/src/types/float/views/index.tsx

```diff
@@ -0,0 +1,122 @@
+/* @jsx jsx */
+
+import { jsx } from '@keystone-ui/core';
+import { FieldContainer, FieldLabel, TextInput } from '@keystone-ui/fields';
+import { CellLink, CellContainer } from '@keystone-next/admin-ui/components';
+
+import {
+  CardValueComponent,
+  CellComponent,
+  FieldController,
+  FieldControllerConfig,
+  FieldProps,
+} from '@keystone-next/types';
+
+export const Field = ({ field, value, onChange, autoFocus }: FieldProps<typeof controller>) => (
+  <FieldContainer>
+    <FieldLabel>{field.label}</FieldLabel>
+    {onChange ? (
+      <TextInput
+        autoFocus={autoFocus}
+        onChange={event => onChange(event.target.value)}
+        value={value}
+      />
+    ) : (
+      value
+    )}
+  </FieldContainer>
+);
+
+export const Cell: CellComponent = ({ item, field, linkTo }) => {
+  let value = item[field.path] + '';
+  return linkTo ? <CellLink {...linkTo}>{value}</CellLink> : <CellContainer>{value}</CellContainer>;
+};
+Cell.supportsLinkTo = true;
+
+export const CardValue: CardValueComponent = ({ item, field }) => {
+  return (
+    <FieldContainer>
+      <FieldLabel>{field.label}</FieldLabel>
+      {item[field.path]}
+    </FieldContainer>
+  );
+};
+
+export const controller = (config: FieldControllerConfig): FieldController<string, string> => {
+  return {
+    path: config.path,
+    label: config.label,
+    graphqlSelection: config.path,
+    defaultValue: '',
+    deserialize: data => (parseFloat(data[config.path]) || '') + '',
+    serialize: value => ({ [config.path]: parseFloat(value.replace(',', '.')) || null }),
```

Comment:
This logic should be identical to the logic in integer except it should call parseFloat

Reply:
In this logic I wanted to add support for `,` as decimal separator together with `.`, because it is popular in many countries
https://en.wikipedia.org/wiki/Decimal_separator#Countries_using_decimal_comma
So pasting something like `3,14159` should be converted to `3.14159` instead of `314159` or `3`.
But this is debatable change, so now let's keep the previous logic to merge this, I've reverted this line to integer logic.